src/Credit/Payment.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Wearesho\Bobra\Portmone\Credit;
+
+use Wearesho\Bobra\Payments;
+
+/**
+ * Class Payment
+ * @package Wearesho\Bobra\Portmone\Credit
+ */
+class Payment implements Payments\Credit\Response
+{
+    /** @var string */
+    protected $billId;
+
+    /** @var \DateTimeInterface */
+    protected $payedDate;
+
+    /** @var float */
+    protected $payedAmount;
+
+    /** @var string */
+    protected $status;
+
+    /**
+     * Payment constructor.
```

Comment:
phpDoc for constructor should not be generated.

Reply:
fix